spec/usermanual/admins.html.erb

```diff
@@ -0,0 +1,243 @@
+  </p>
+
+  <h2>Accepting pending organizations</h2>
+
+  <p>
+    Another responsibility for administrators is to manage pending organization requests, accepting or denying them
+    as they see fit. Anyone can make an organization request, so they should be accepted through careful consideration.
+  </p>
+
+  <p>Administrator will see this notice on the front page when a new organization is requested:</p>
+
+<%
+  FactoryGirl.create :accepted_organization, name: 'Some organization', information: 'Some information'
+  FactoryGirl.create :user, login: 'user', password: 'user'
+
+  click_on 'Sign out'
+  fill_in 'session_login', with: 'user'
+  fill_in 'session_password', with: 'user'
+  click_on 'Sign in'
+
+  visit '/'
+  click_link 'Request a new organization'
+  fill_in 'organization[name]', with: 'Helsingin yliopisto'
+  fill_in 'organization[information]', with: 'University life'
+  fill_in 'organization[slug]', with: 'hy'
+  click_button 'Request organization'
+  expect(page).to have_content 'Organization was successfully requested.'
+
+  click_on 'Sign out'
+  fill_in 'session_login', with: 'admin'
+  fill_in 'session_password', with: 'admin'
+  click_on 'Sign in'
+  visit '/'
+  highlight 'a:contains("Show")'
+%>
+<%= screenshot %>
+
+  <p>
+    Clicking 'show' will list all pending organization requests.
+  </p>
+
+<%
+  click_on 'Show'
+  highlight 'a:contains("Accept")'
+%>
+<%= screenshot %>
+<%
+  click_on 'Accept'
+  expect(page).to have_content 'Organization request was successfully accepted.'
+%>
+<%= screenshot %>
+
+  <p>
+    After clicking 'Accept', organization requester is promoted as organization's teacher and obtains rights to manage it.
+  </p>
+<%
+  visit '/'
+%>
+<%= screenshot %>
+
+<p>
+  Alternatively, after clicking 'Reject', administrator must fill a form to give a reason for rejection.
+</p>
+
+<%
+  Organization.find_by(slug: 'hy').destroy!
+  click_on 'Sign out'
+  fill_in 'session_login', with: 'user'
+  fill_in 'session_password', with: 'user'
+  click_on 'Sign in'
+
+  visit '/'
+  click_link 'Request a new organization'
+  fill_in 'organization[name]', with: 'Helsingin yliopisto'
+  fill_in 'organization[information]', with: 'University life'
+  fill_in 'organization[slug]', with: 'hy'
+  click_button 'Request organization'
+  expect(page).to have_content 'Organization was successfully requested.'
+
+  click_on 'Sign out'
+  fill_in 'session_login', with: 'admin'
+  fill_in 'session_password', with: 'admin'
+  click_on 'Sign in'
+  visit '/'
+
+  click_on 'Show'
+  highlight 'a:contains("Reject")'
+%>
+<%= screenshot %>
+
+<%
+   click_on 'Reject'
+   fill_in 'organization_rejected_reason', with: 'No such university exists, you can\'t fool me!'
+%>
+<%= screenshot %>
+<%
+  click_on 'Reject organization'
+  expect(page).to have_content 'Organization request was successfully rejected.'
+%>
+<%= screenshot %>
+
+<p>Requested organization will not be shown anywhere in the front page after rejection.</p>
```

Comment:
We have a user story in backlog about organization reject notice reaching the organization requester